protobufs/entity/service.proto

```diff
@@ -74,7 +94,13 @@ message EntityUpdatePayload {
 
 message EntityResponse {
   document.ResponseHeader header = 1;
-  EntityData data = 2;
+  EntityWithReleationships data = 2;
+}
+
+message Relationship {
+  string identity = 1;
+  bool active = 2;
+  string last_update_at_entity_version = 3;
```

Comment:
Can you explain this field?

Reply:
the document entity timestamp we discussed. 

I would like to add it to the protobuf/entityrelationship too.

Everytime a entityrelationship gets modified we update it.

Reply:
I think we don't need it. We could use the block.height from the `anchorRegistry`. In case the information is needed.